lib/broadway/server.ex

```diff
@@ -48,6 +53,17 @@ defmodule Broadway.Server do
     {:reply, state.producers_names, state}
   end
 
+  def handle_call({:update_rate_limiting, opts}, _from, state) do
+    %{rate_limiting?: rate_limiting?, name: name} = state
+
+    if rate_limiting? do
+      reply = Broadway.RateLimiter.update_rate_limiting(name, opts)
```

Comment:
Instead of doing a blocking call here, have this function return the rate_limiting name (as mentioned above).

Reply:
Done.